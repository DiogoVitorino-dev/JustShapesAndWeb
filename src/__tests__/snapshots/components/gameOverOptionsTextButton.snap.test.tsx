import { render } from "test-utils";

import OptionsTextButton from "@/components/game/gameOver/optionsTextButton";

describe("Game Over Options Text Button - Snapshot component test", () => {
  it("should render correctly", () => {
    const tree = render(<OptionsTextButton />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
