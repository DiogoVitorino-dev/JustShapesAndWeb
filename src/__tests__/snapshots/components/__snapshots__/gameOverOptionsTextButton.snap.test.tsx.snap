// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game Over Options Text Button - Snapshot component test should render correctly 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
